configure.ac

```
@@ -619,6 +661,14 @@ AC_ARG_ENABLE([mmap-map-stack],
   [AS_HELP_STRING([--enable-mmap-map-stack],
     [use mmap to allocate stacks instead of malloc])])
 
+AC_ARG_WITH([relative-libdir],
+  [AS_HELP_STRING([--with-relative-libdir],
+    [location of the Standard Library, specified relative to --bindir])],
+  [AS_IF([test x"$withval" = 'xno'],
+    [bindir_to_libdir=''],
```

A quality-of-life suggestion (that could wait for later):
```suggestion
  [AS_CASE([$withval],
    [no],
      [bindir_to_libdir=''],
    [yes],
      [bindir_to_libdir=../lib/ocaml],
```
(it would also need a nested `AS_CASE` to yield a Windows path when applicable in the `yes` case, I suppose).
Noticed this while building a cross-compiler to Windows with relative libdir, which seemed to work as expected (ie I cannot rename the directory in which it is installed, but IIUC only the combination with the other PRs would make that work).

---

This is a brilliant idea, thanks! It gets rid of the two cases in the opam file, too (I absolutely want the backslashes on Windows, but I didn't like having to do that so explicitly in the opam file)